### PR TITLE
Bids 1544/show inactive validators as inactive

### PR DESCRIPTION
**Pull request by Eisei24**
You can check with https://prater.beaconcha.in/dashboard?validators=11877,55293,222906,255562,311255,453473 but the last one might no longer be pending by that time.

### CODE REVIEW

File: templates/dashboard.html [hunk 1]
```diff
@@ -229,7 +229,7 @@
                   </div>
                 </span>
                 <div style="font-weight:300;font-size:1rem;display:flex;justify-content:center" class="pb-2">
-                  <span id="dashboard-info">Found 0 pending, 0 active and 0 exited validators</span>
+                  <span id="dashboard-info">0 active (0 online, 0 offline), 0 pending, 0 exited validators</span>
```
Comment (Eisei24) [inline templates/dashboard.html:232]: Afaik this is just a dummy text that just serves to illustrate what the formatting looks like.

Comment (D13ce) [reply templates/dashboard.html:232]: This is not entirely true, this text is shown as long as the dashboard is loading. I surfed to the link you sent to test your changes and saw said text for ~2s.

**Review by D13ce — COMMENTED** [review]
Your changes look good but slashed validators are not part of the text. I have asked @Buttaa in Jira about this, let's see what he says.

File: templates/dashboard.html (hunk 1 above)
Comment (Eisei24) [reply templates/dashboard.html:232]: I see, thanks for the info.

**Comment by Eisei24** [comment]
Slashed validators are now also written if we have some (no "0 slashed").

**Review by D13ce — APPROVED** [review]
lgtm

**Pull request merged**